Dockerfile

```diff
@@ -1,10 +1,10 @@
 FROM golang:1.10.4-alpine as builder
 
 WORKDIR /go/src/github.com/dragonflyoss/Dragonfly
-COPY . /go/src/github.com/dragonflyoss/Dragonfly
-
 RUN apk --no-cache add bash make gcc libc-dev git
 
+COPY . /go/src/github.com/dragonflyoss/Dragonfly

```

Comment:
@cd1989 Maybe there is something that I don't know. Could you please tell me that what's the difference between them?

Reply:
`COPY . /go/src/github.com/dragonflyoss/Dragonfly` layer always change, but layer `RUN apk --no-cache add bash make gcc libc-dev git` not. If this static layer is put after the dynamic one, it can't take advantage of cache of the static layer.